includes/orphaned_objects.inc

```diff
@@ -0,0 +1,262 @@
+<?php
+
+/**
+ * @file
+ * A list of orphaned Islandora objects.
+ */
+
+/**
+ * Builds the Orphaned Islandora Objects management form.
+ *
+ * @param array $form
+ *   An array representing a form within Drupal.
+ * @param array $form_state
+ *   An array containing the Drupal form state.
+ *
+ * @return array
+ *   An array containing the form to be rendered.
+ */
+function islandora_manage_orphaned_objects_form($form, $form_state) {
+  if (isset($form_state['show_confirm'])) {
+    $pids = $form_state['pids_to_delete'];
+    if (count($form_state['pids_to_delete']) <= 10) {
+      $pid_paths = '<ul>';
+      foreach ($pids as $pid) {
+        $pid_paths = $pid_paths . '<li><a href="/islandora/object/' . $pid . '" target="_blank">' . $pid . '</a></li>';
```

Comment:
This link will not be multisite or langauge safe. It is advisable to use the l() function.

Reply:
Addressed! (Though I am curious about what you mean by multisite safe.)

Reply:
Multisites can share domains like so: `www.example.com/my_site`.  The `l` function accounts for this as it is wired through the url function that uses the base_url and accounts for many things.